sources/local_settings_example.py

```diff
@@ -12,16 +12,16 @@
 from local_settings import *
 
 sys.path.insert(0, SEFARIA_PROJECT_PATH)
-os.environ['DJANGO_SETTINGS_MODULE'] = "settings"
```

Comment:
`"local_settings"` seemed like a better default example than `"settings"` because that is what the file is called

but totally acknowledge that this file generally shouldn't be changed